Update notification if there's already one open.

I'm running Elementary OS (Loki) and whenever I skipped a song, a new notification would open.  I would have a bunch of notifications open and it was annoying.  So I added a tag to the notification object, so now it will update the notification if there's one open.  If other people are interested in this I can make a pull request.

As a side note, when I ran npm run lint, I got this:

npm ERR! Linux 4.10.0-33-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "lint"
npm ERR! node v6.11.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! museeks@0.8.1 lint: `eslint . --config .eslintrc --cache`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the museeks@0.8.1 lint script 'eslint . --config .eslintrc --cache'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the museeks package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint . --config .eslintrc --cache

--- a/src/ui/actions/NotificationActions.js
+++ b/src/ui/actions/NotificationActions.js
@@ -11,6 +11,7 @@ const add = (notificationData) => {
           body: notificationData.body,
           icon: notificationData.icon,
           silent: true,
+          tag: 'onlyOneNotification',
         }
       );
 

--- a/src/ui/actions/NotificationActions.js
+++ b/src/ui/actions/NotificationActions.js
@@ -11,7 +11,7 @@ const add = (notificationData) => {
           body: notificationData.body,
           icon: notificationData.icon,
           silent: true,
-          tag: 'onlyOneNotification',
+          tag: 'museeks-notification',
         }
       );
 

